leaderboard/enriching/instrumented_enricher.go

```diff
@@ -39,12 +38,9 @@ func (en *instrumentedEnricher) Enrich(ctx context.Context, tenantID, leaderboar
 	})
 	defer span.Finish()
 
-	members, err := en.impl.Enrich(ctx, tenantID, leaderboardID, members)
+	res, err := en.impl.Enrich(ctx, tenantID, leaderboardID, members)
```

Comment:
Why this?

Reply:
You mean the rename? No reason really, I'll remove it.